lara-typescript/package.json

```diff
@@ -86,8 +89,9 @@
     "node-sass": "^4.12.0",
     "npm-run-all": "^4.1.5",
     "raw-loader": "^1.0.0",
-    "react": "^16.8.6",
-    "react-dom": "^16.8.6",
+    "react": "^16.13.1",
+    "react-dom": "^16.13.1",
```

Comment:
There is also the react version provided by LARA which is older.
If we bump the version here, then it will be easy for developers to use react features that aren't compatible with that older version in LARA.

Doug thought it wouldn't be easy to update the older version in LARA because all of the ITSI authoring page is written with this older version of React (and probably coffeescript). 

I'm not sure how to handle this.

Reply:
Perhaps this version bump of React won't break the itsi pages. Based on the version numbering that should be true. So perhaps that is the best approach. Bump it for all of LARA and then we can do some smoke testing of the ITSI authoring.

cc @dougmartin